src/frontend/src/utils/gantt.utils.ts

```diff
@@ -13,16 +27,20 @@ export const NO_TEAM = 'No Team';
 export const GANTT_CHART_GAP_SIZE = '0.75rem';
 export const GANTT_CHART_CELL_SIZE = '2.25rem';
 
-export type TaskType = 'task' | 'milestone' | 'project';
 export interface GanttTaskData {
   id: string;
-  type: TaskType;
   name: string;
   start: Date;
   end: Date;
-  workPackages: GanttTaskData[];
   projectNumber: number;
   carNumber: number;
+  workPackageNumber: number;
+  totalWorkPackages: WorkPackage[];
+  unblockedWorkPackages: WorkPackage[];
+  blocking: WbsNumber[];
```

Comment:
why blocking here and immediately blocking elsewhere?

Reply:
good question.